web/src/lib/components/admin-page/settings/admin-settings.svelte

```diff
@@ -25,16 +26,17 @@
     }
   };
 
-  const handleSave = async (config: Partial<SystemConfigDto>) => {
+  const handleSave = async (update: Partial<SystemConfigDto>) => {
     try {
-      const result = await api.systemConfigApi.updateConfig({
+      const { data: newConfig } = await api.systemConfigApi.updateConfig({
         systemConfigDto: {
           ...savedConfig,
-          ...config,
+          ...update,
         },
       });
 
-      savedConfig = { ...result.data };
+      config = cloneDeep(newConfig);
```

Comment:
This line makes it possible so that if the response from " update config" is different than the current config, to display it immediately. Saving "accepted codecs" that doesn't include the target codec, will automatically mark it as accepted after saving.